basculechecks/provide.go

```diff
@@ -45,7 +48,7 @@ func ProvideCapabilitiesMapValidator(key string) fx.Option {
 			arrange.UnmarshalKey(key, CapabilitiesMapConfig{}),
 			NewCapabilitiesMap,
 		),
-		ProvideMetricValidator(),
+		ProvideMetricValidator(false),
```

Comment:
I couldn't quite catch why the capabilities component is required in `ProvideCapabilitiesMapValidator` but it is optional in `ProvideRegexCapabilitiesValidator`